src/ts/jsoneditor.ts

```diff
@@ -68,27 +67,36 @@ $(document).ready(function() {
           }
         }
       });
-      currentFileUploadID = $(this).data('id');
       currentFileItemID = $(this).data('uploadid');
     });
 
-    $(document).on('click', '.jsonSaver', function(){
-      if (typeof currentFileUploadID === 'undefined') {
+    // Clear the JSONEditor and unload the file
+    $(document).on('click', '.jsonClear', function() {
+      currentFileItemID = undefined;
+      editor.set({});
+      $('#jsonEditorTitle').html('File was unloaded.');
+    });
+
+    // The save function is now defined separately
+    const saveJsonFile = function(){
+      if (typeof currentFileItemID === 'undefined') {
         // we are creating a new file
         const realName = prompt('Enter name of the file');
         if (realName == null) {
           return;
         }
-        const id = $('#main_form').find('input[name="id"]').attr('value');
+        // add the new name for the file as a title
+        $('#jsonEditorTitle').html('Filename: ' + realName + '.json');
```

Comment:
The "Filename" string (without `:`) should be i18n. See https://github.com/elabftw/elabdoc/blob/next/doc/contributing.rst#adding-a-new-term-for-js-i18n

Also, it might be good to check if the input doesn't end in .json to avoid having two .json if the user enters the extension too.

Apart from that, it seems to work fine :+1:

Reply:
Should I translate the term in all languages? Do you just accept what Google translate offers or some other translation service? I can verify a few language terms from natives. But you do support a number of languages.

Reply:
Oh yeah, I use google translate, for something as "Filename" you should have no issue. I know it's tedious, but right know it's like that. I can also do it if you don't feel like copy pasting 17 times some stuff, I have a good setup allowing me to do that without too much hassle.

Reply:
I have done most. Please check pt_PT though. I believe Google only does br_PT. But I will just copy paste the same there and you can have a look.

Reply:
Yeah, I just take portuguese for pt_br.